src/styles/tokens/colors.scss

```
@@ -1,43 +1,113 @@
+$color-input: #f0f0ef;
+$color-surface: #ffffff;
+$color-surface-muted: $color-bg-secondary;
+$color-surface-soft: $color-input;
+$color-surface-elevated: #ffffff;
+$color-surface-strong: $color-bg-muted;
+
+$color-card: $color-surface;
+$color-card-foreground: $color-text;
+
+// ── Interaction ──
+$color-hover: mix($color-primary, #ffffff, 8%);
```

Stylelint (17.5.0)

[error] Expected color.mix instead of mix (scss/no-global-function-names)

(scss/no-global-function-names)